plotly/colors.py

```diff
@@ -520,6 +542,10 @@ def n_colors(lowcolor, highcolor, n_colors):
                      lowcolor[2] + (index * incr_2))
         color_tuples.append(new_tuple)
 
+    if colortype == 'rgb':
+        # convert back to rgb
+        color_tuples = color_parser(color_tuples, label_rgb)
```

Comment:
It should, but I kept it like that so that `color_tuples` is a common variable in both paths the function can take. Shall I change to `list_of_colors` for an apt name?